### PR TITLE
feat: set default timeout 20s for outgoing requests in SSR

### PR DESCRIPTION
**Pull request by Platonn**
Set a default timeout for outgoing http calls in SSR to 20s.

Note: the timeout mechanism was introduced before in PR https://github.com/SAP/spartacus/pull/16658

closes https://jira.tools.sap/browse/CXSPA-2234

### CODE REVIEW

**Review by bgambocjaviniar — APPROVED** [review]

File: projects/core/src/http/http-timeout/default-http-timeout.config.ts [hunk 1]
```diff
@@ -9,8 +9,7 @@ import { OccConfig } from '../../occ/config/occ-config';
 export const defaultBackendHttpTimeoutConfig: OccConfig = {
   backend: {
     timeout: {
-      // [CXSPA-2234] (breaking change): uncomment it only in the next major (v6.0?):
-      // server: 10_000,
+      server: 10_000,
```
Comment (bgambocjaviniar) [inline projects/core/src/http/http-timeout/default-http-timeout.config.ts]: shouldn't it be 20_000 for 20s

**Review by bgambocjaviniar — CHANGES_REQUESTED** [review]
see comment

File: projects/core/src/http/http-timeout/default-http-timeout.config.ts (hunk 1 above)
Comment (hackergil) [reply projects/core/src/http/http-timeout/default-http-timeout.config.ts]: I think so ;)

Comment (Platonn) [reply projects/core/src/http/http-timeout/default-http-timeout.config.ts]: 😅 Initially I thought 10s, but then decided to be more generous with 20s. However it seems I didn’t commit my change. 🤦 
Now pushed update 20_000.

**Review by bgambocjaviniar — APPROVED** [review]

**Pull request merged**